### DISCUSSION TITLE
Spark/MXnet: Logic of Code

### DISCUSSION THREAD

**Discussion by luzengxiang** [apache/incubator-mxnet#12765] [Q&A]
vVersion: 1.3.0

Spark/MXnet.scala: 
class MXNet ,
method: setFeedForwardModel

Line:180
epochSize = numExamples / params.batchSize / kv.numWorkers

The value of this parameter is confusing. Is it supposed to be numExamples / (params.batchSize / kv.numWorkers)?

**Comment by piyushghai** [comment]
@mxnet-label-bot [scala, Question]

**Comment by piyushghai** [comment]
@luzengxiang : For questions, please submit on MXNet discussion forum (https://discuss.mxnet.io), where it will get a wider audience and allow other to learn as well.

**Comment by lanking520** [comment]
@CodingCat @yzhliu @nswamy For spark stuff

**Comment by luzengxiang** [comment]
Class: org.apache.mxnet.spark.MXNet
